crates/subspace-farmer/src/commitments.rs

```diff
@@ -132,8 +132,11 @@ impl Commitments {
                     .map(|piece| subspace_solving::create_tag(piece, salt))
                     .collect();
 
-                for (tag, index) in tags.iter().zip(batch_start..) {
-                    db.put(tag, index.to_le_bytes())
+                for (tag, offset) in tags
+                    .iter()
+                    .zip(batch_start..batch_start + pieces_to_process)
```

Comment:
This is simpler:
```suggestion
                    .zip(batch_start..)
```

Zip will only take the number of items that corresponds to the smaller of two iterators, so no reason to specify size for the second.